src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+//unable to update duke.txt after marking, unmarking or deleting in this version
+import jdk.jshell.Snippet;
+import java.io.*;
+import java.time.chrono.MinguoDate;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static int IND_COUNT;
+    private static Task[] taskList;
+
+    private static void appendToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+   //adds previous tasks back into taskList when restarting Duke
```

Comment:
```suggestion
   // adds previous tasks back into taskList when restarting Duke
```

There appears to be a [missing whitespace](https://se-education.org/guides/conventions/java/intermediate.html#layout) here. It would also be good to write this as [a JavaDoc](https://se-education.org/guides/conventions/java/intermediate.html#comments) instead.